thinc/backends/ops.py

```diff
@@ -603,7 +603,7 @@ def dtanh(self, Y: FloatsT, *, inplace: bool = False) -> FloatsT:
             Y += 1.0
             return Y
         else:
-            return 1 - Y ** 2
+            return 1 - Y**2
```

Comment:
Not sure which variant comes from the old version of `black` and which from the new, but it seems like we are flipping between the formats between different PRs 😅.

Reply:
cf https://github.com/explosion/spaCy/pull/10209 --> the change from the PR seems the right "current" `black` format. We should probably pin 
```
black>=22.0,<23.0
```
and make sure that everyone updates their `black` in their local `venv`